### DISCUSSION TITLE
dp convert-from

### DISCUSSION THREAD

**Discussion by chenfan22** [deepmodeling/deepmd-kit#1682] [Q&A] [answered]
Hi, I kept having the error messgae when I tried to use dp convert-from to transfrom a v 1.0 potential. 
The error message is:
WARNING:tensorflow:From /opt/deepmd-kit/lib/python3.9/site-packages/tensorflow/python/compat/v2_compat.py:111: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
WARNING:root:To get the best performance, it is recommended to adjust the number of threads by setting the environment variables OMP_NUM_THREADS, TF_INTRA_OP_PARALLELISM_THREADS, and TF_INTER_OP_PARALLELISM_THREADS.
WARNING:root:Environment variable KMP_BLOCKTIME is empty. Use the default value 0
WARNING:root:Environment variable KMP_AFFINITY is empty. Use the default value granularity=fine,verbose,compact,1,0
/opt/deepmd-kit/lib/python3.9/importlib/__init__.py:169: UserWarning: The NumPy module was reloaded (imported a second time). This can in some cases result in small but subtle issues and is discouraged.
  _bootstrap._exec(spec, module)
WARNING:tensorflow:From /opt/deepmd-kit/lib/python3.9/site-packages/deepmd/utils/convert.py:113: FastGFile.__init__ (from tensorflow.python.platform.gfile) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.gfile.GFile.
DEEPMD WARNING From /opt/deepmd-kit/lib/python3.9/site-packages/deepmd/utils/convert.py:113: FastGFile.__init__ (from tensorflow.python.platform.gfile) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.gfile.GFile.
Traceback (most recent call last):
  File "/opt/deepmd-kit/bin/dp", line 10, in <module>
    sys.exit(main())
  File "/opt/deepmd-kit/lib/python3.9/site-packages/deepmd/entrypoints/main.py", line 489, in main
    convert(**dict_args)
  File "/opt/deepmd-kit/lib/python3.9/site-packages/deepmd/entrypoints/convert.py", line 18, in convert
    convert_20_to_21(input_model, output_model)
  File "/opt/deepmd-kit/lib/python3.9/site-packages/deepmd/utils/convert.py", line 96, in convert_20_to_21
    convert_pb_to_pbtxt(input_model, 'frozen_model.pbtxt')
  File "/opt/deepmd-kit/lib/python3.9/site-packages/deepmd/utils/convert.py", line 116, in convert_pb_to_pbtxt
    tf.import_graph_def(graph_def, name='')
  File "/opt/deepmd-kit/lib/python3.9/site-packages/tensorflow/python/util/deprecation.py", line 552, in new_func
    return func(*args, **kwargs)
  File "/opt/deepmd-kit/lib/python3.9/site-packages/tensorflow/python/framework/importer.py", line 407, in import_graph_def
    return _import_graph_def_internal(
  File "/opt/deepmd-kit/lib/python3.9/site-packages/tensorflow/python/framework/importer.py", line 503, in _import_graph_def_internal
    results = c_api.TF_GraphImportGraphDefWithResults(
tensorflow.python.framework.errors_impl.NotFoundError: Op type not registered 'DescrptNorot' in binary running on compute1-exec-136.ris.wustl.edu. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.

The version is 2.1.1_cpu from Docker image. The command I used is : dp convert-from -i frozen_model.pb -o newpot 2.0

Could you please give me some suggestion ? Thank you for the help.
Fangzheng

**Comment by njzjz** [comment, marked as answer]
This is a v0.12 potential, which is not supported to convert from yet.

**Reply by chenfan22** [reply]
Thank you for the explantion.

Best
Fangzheng
